libraries/abstractions/common_io/test/test_iot_spi.c

```diff
@@ -995,11 +1011,12 @@ TEST( TEST_IOT_SPI, AFQP_IotSPI_TransferAsyncAssisted )
     uint8_t ucTxBuf[ 8 ] = { 0 };
     char cMsg[ 50 ] = { 0 };
     int32_t lLoop = 0;
+    int i = 0, len = 0;
 
     srand( xTaskGetTickCount() );
 
     /* Generate random tx bytes and load them to a string to print later. */
-    for( int i = 0, len = sizeof( ucTxBuf ); i < len; i++ )
+    for( i = 0, len = sizeof( ucTxBuf ); i < len; i++ )
```

Comment:
Do we for sure want randomness in this test? If so, unless we're seeding rand() with something like clock time, I bet that the stream is the same each time.

Reply:
I think we do. The tests want to send random bytes to external device.
See the line above, we already have srand( xTaskGetTickCount() )